lib/ruby_lsp/handler.rb

```diff
@@ -227,7 +227,7 @@ def telemetry_params(request, request_time, error)
         params[:errorMessage] = error.message
       end
 
-      params[:uri] = uri if uri
+      params[:uri] = uri.sub(%r{.*://#{Dir.home}}, "~") if uri
```

Comment:
What if I open the source of a gem located in `/opt/rubies/`? 

I wonder if we should restrict the metrics to files inside the project or something like this?